config/initializers/devise.rb

```diff
@@ -3,7 +3,7 @@
 # Use this hook to configure devise mailer, warden hooks and so forth.
 # Many of these configuration options can be set straight in your model.
 Devise.setup do |config|
-  config.secret_key = 'dabd54db3b77f2b880cffc1e90c85270063db8c636eb3280dd0c81bf40f633c3901a5c3d0ec04cb5965c2ab9aceac8e39ea08ad6b7dd3f431d2b4651080e586b'
+  config.secret_key = ENV.fetch('DEVISE_SECRET_KEY')
```

Comment:
I have already generated new values for this secret (by running `bundle exec rails secret` multiple times) and put those values into the env for staging and prod. When the new secrets are activated after restart then any existing password resets in the DB will need to be triggered again. Given that AFAIK **nobody** logs in to the admin interface these days this shouldn't disrupt anything.